jupyterlab_git/git.py

```diff
@@ -8,6 +8,7 @@
 
 import pexpect
 import tornado
```

Comment:
```suggestion
import tornado
import tornado.locks
```

It looks like the problem in the test is being caused by the fact that `locks` is a seperate module that doesn't get imported by default on `import tornado`. If you're familiar with scipy, it's just like how you would need to do `import scipy.stats` before you can actually reference `scipy.stats` (or whichever scipy module)